clients/roscpp/include/ros/steady_timer.h

```diff
@@ -67,9 +67,11 @@ class ROSCPP_DECL SteadyTimer
 
   /**
    * \brief Set the period of this timer
+   * \param reset Whether to reset the timer. If true, timer ignores elapsed time and next cb occurs at now()+period
    */
   void setPeriod(const WallDuration& period, bool reset=true);
 
+  bool hasStarted() const { return impl_->hasStarted(); }
```

Comment:
The corresponding `Impl::hasStarted()` seem to be still missing in steady_timer.cpp

Reply:
Thanks, fixed in https://github.com/ros/ros_comm/pull/1557/commits/c37a75bedac9945b640c10c83e5609625e746cbc.